examples/pytorch/rgcn/experimental/README.md

```diff
@@ -156,6 +156,10 @@ More details about the four steps are explained in our
 The graph structure should be written as a node file and an edge file. The node features and edge features
 can be written as DGL tensors. `write_mag.py` shows an example of writing the OGB MAG graph into files.
 
+As `pm_dglpart` cannot handle self-loops and duplicate edges correctly, these edges are removed and stored
+into `mag_removed_edges.txt`. When converting ParMETIS outputs into DGLGraph in next steps, the file that
```

Comment:
i suppose write_mag.py writes the removed edges to mag_removed_edges.txt?

Reply:
if so, please mention it in the readme.

Reply:
yes, this added sentences lies in `write_mag.py` part.